OBSDOCS-1751 incident detection ui plugin (#90113)
openshift/openshift-docs: modules/coo-incident-detection-using.adoc

// Module included in the following assemblies:

// * observability/cluster_observability_operator/ui_plugins/incident-detection-ui-plugin.adoc

:_mod-docs-content-type: PROCEDURE
[id="coo-incident-detection-using_{context}"]
= Using {coo-full} incident detection

.Prerequisites

* You have access to the cluster as a user with the `cluster-admin` cluster role.
* You have logged in to the {product-title} web console.
* You have installed the {coo-full}.
* You have installed the {coo-full} monitoring UI plugin with incident detection enabled.

.Procedure

. In the Administrator perspective of the web console, click on *Observe* -> *Incidents*.

> . The Incidents Timeline UI shows the grouping of alerts into *incidents*. The color coding of the lines in the graph corresponds to the severity of the incident.

Reviewer: Add a note that once the feature is enabled, it will take at least 10 minutes to process the correlations before you see the timeline.

Reviewer: In addition, the analysis and grouping into incidents is performed only for alerts that are firing after feature enablement. Alerts resolved prior to feature enablement are not included.

+
image::coo-incidents-timeline.png[]

. By clicking on an incident, you can see the timeline of alerts that are part of that incident, in the Alerts Timeline UI.
+
image::coo-incident-alerts-timeline.png[]

> . In the list of alerts, alerts are mapped to specific components, which are ranked by criticality.

Reviewer: ```suggestion
. In the list of alerts, alerts are mapped to specific components, which are grouped by severity.
```

+
image::coo-incident-alerts-components.png[]

. Click to expand a compute component in the list. The underlying alerts related to that component are displayed.
+
image::coo-incident-alerts-components-expanded.png[]

> . Click on an alert link to see detailed information about that alert.

Reviewer: Please note that this is only available for firing alerts at this time.

+
image::coo-incidents-timeline-weekly.png[]
+
[NOTE]
====
It will take at least 10 minutes to process the correlations and to see the timeline, after you enable incident detection.

The analysis and grouping into incidents is performed only for alerts that are firing after you have enabled this feature. Alerts that have been resolved before feature enablement are not included.
====

. Zoom in to a 1-day view by clicking on the drop-down to specify the duration.
+
image::coo-incidents-timeline-daily.png[]

. By clicking on an incident, you can see the timeline of alerts that are part of that incident, in the Alerts Timeline UI.
+
image::coo-incident-alerts-timeline.png[]

. In the list of alerts that follows, alerts are mapped to specific components, which are grouped by severity.
+
image::coo-incident-alerts-components.png[]

. Click to expand a compute component in the list. The underlying alerts related to that component are displayed.
+
image::coo-incident-alerts-components-expanded.png[]

> . Click on the link for a firing alert, to see detailed information about that alert.

Reviewer 2: ```suggestion
. Click the link for a firing alert, to see detailed information about that alert.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c

[NOTE]
====
**Known issues**

> * Depending on the order of the timeline bars, the tooltip may overlap and hide the underlying bar. You can still click the bar and select the incident or alert.

Reviewer 2: ```suggestion
* Depending on the order of the timeline bars, the tooltip might overlap and hide the underlying bar. You can still click the bar and select the incident or alert.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m
